crates/bitwarden-uniffi/src/auth/mod.rs

```diff
@@ -76,4 +78,16 @@ impl ClientAuth {
             .auth()
             .make_register_keys(email, password, kdf)?)
     }
+
+    /// Validate the user password
+    pub async fn validate_password(&self, password: String) -> Result<bool> {
+        Ok(self
+            .0
+             .0
+            .write()
+            .await
+            .auth()
+            .validate_password(password, "".to_string())
```

Comment:
This function should also take the password hash as a parameter, as it is, it will always fail because the hash is never going to be the empty string

Reply:
Correct, seems I forgot to fix it 🤦